cpp/log/cert_checker.cc

```diff
@@ -152,7 +152,7 @@ Status CertChecker::CheckIssuerChain(CertChain* chain) const {
   // because we will later check that it is trusted.
   Status status = chain->IsValidCaIssuerChainMaybeLegacyRoot();
   if (!status.ok()) {
-    LOG(ERROR) << "Failed to check issuer chain";
+    LOG(ERROR) << "Failed to check issuer chain: " << status;
```

Comment:
The string? `status` has an `operator<<` that will invoke `ToString()` on it, which looks like this:

https://github.com/google/certificate-transparency/blob/master/cpp/util/status.cc#L70